api/specificAccessOrchastrator/index.spec.ts

```diff
@@ -4,21 +4,23 @@ import 'mocha';
 import * as sinon from 'sinon';
 import * as sinonChai from 'sinon-chai';
 import { mockReq, mockRes } from 'sinon-express-mock';
+import { getTaskType, orchestrationSpecificAccessRequest, postCreateTask, specificAccessRequestUpdateAttributes } from '.';
```

Comment:
Why has this line been moved from below? Should be the last line as it is imported from same folder